### PR TITLE
add cssgram v0.1.3 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/una/CSSgram
**Watch 73
  Star 3,119
 Fork 173**
@Amomo could u help me check this PR for #6227 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/cssgram/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "CSS",
+    "library",
+    "photo",
+    "effects"
+  ],
+  "homepage": "https://github.com/una/CSSgram",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/una/CSSgram.git",
+    "basePath": "source",
```
Comment (PeterDaveHello) [inline ajax/libs/cssgram/package.json]: Wrong path

**Comment by maruilian11** [comment]
@PeterDaveHello sorry, i have corrected the path field.

**Comment by Amomo** [comment]
@maruilian11 
Sorry for late, I think this PR is fine, could you rebased it to latest branch?
Thank you! :-)

**Comment by maruilian11** [comment]
@Amomo i have rebased it ~

**Comment by maruilian11** [comment]
ping @Amomo

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? thanks! :-D

**Comment by PeterDaveHello** [comment]
No description?

**Comment by maruilian11** [comment]
@PeterDaveHello i have added it.

**Comment by PeterDaveHello** [comment]
@LeaYeh please double check this PR, thanks.

**Comment by maruilian11** [comment]
ping @LeaYeh

**Pull request merged**